cmd/goal/node.go

```diff
@@ -119,18 +120,49 @@ var nodeCmd = &cobra.Command{
 	},
 }
 
+func getMissingCatchpointLabel(URL string) (label string, err error) {
+	resp, err := http.Get(URL)
+	if err != nil {
+		return
+	}
+	if resp.StatusCode != 200 {
+		err = errors.New(resp.Status)
+		return
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return
+	}
+	label = string(body)
+	label = strings.TrimSuffix(label, "\n")
+	return
```

Comment:
Looks like you can check that the catchpoint label can be parsed with `ledgercore.ParseCatchpointLabel`